test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorHostTest.cs

```diff
@@ -47,6 +47,39 @@ public class MvcRazorHostTest
             Assert.Equal("src/file.cshtml", chunkInheritanceUtility.InheritedCodeTreePagePath, StringComparer.Ordinal);
         }
 
+        [Theory]
+        [InlineData("//")]
+        [InlineData("C:/")]
+        [InlineData(@"\\")]
+        [InlineData(@"C:\")]
+        public void DecorateCodeBuilder_DesignTimeRazorPathNormalizer_NormalizesChunkInheritanceUtilityPaths(
+            string rootPrefix)
+        {
+            // Arrange
+            var rootedAppPath = $"{rootPrefix}SomeComputer/Location/Project/";
```

Comment:
Ah, as of right I don't believe that can ever happen as far as tooling is concerned. What scenario did you envision causing that behavior?